src/gps/launch/basestation.launch.py

```diff
@@ -0,0 +1,30 @@
+import launch
+import launch_ros.actions
+
+
+def generate_launch_description():
+    return launch.LaunchDescription(
+        [
+            launch_ros.actions.Node(
+                package="gps",
+                executable="rtcm_pub_node",
+                name="gps_basestation_node",
+                parameters=[
+                    {"TimingMode": 1},  # Survey In mode
+                    {"MinTime": 60},  # Survey in time (s)
+                    {"MinAcc": 10.0},  # Survey In minimum accuracy (m)
+                    {"Freq": 2.0},  # Publish rate (hz)
+                    {"Baudrate": 9600},
```

Comment:
default baudrate is 9600 here but different in the IOManager?

Reply:
The IOManager has a default of 38XXXX which was the default baudrate of the f9p UART1 pin. That default value never gets used as this parameter value gets passed in so it doesn't matter. 9600 is the default baudrate for the USB interface. I can set the IOManager (or whatever it gets named) to 9600 when I move it since we will only use the USB interface on the base station side.